lib/helper/Appium.js

```diff
@@ -1351,6 +1351,15 @@ class Appium extends Webdriver {
     return super.grabValueFrom(parseLocator.call(this, locator));
   }
 
+  /**
+   * {{> scrollIntoView }}
+   *
+   * Supported only for web testing
+   */
+  async scrollIntoView(locator, scrollIntoViewOptions) {
```

Comment:
If `scrollIntoView` may be used only in web, why add this method for Appium? 
Maybe delete this method from Appium?

Reply:
"web testing" means Appium web testing here @voro